source/code-snippets/find.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func main() {
+	// Replace the uri string with your MongoDB deployment's connection string.
+	uri := "mongodb+srv://<username>:<password>@<cluster-address>/test?w=majority"
+
+	ctx := context.TODO()
+	client, err := mongo.Connect(ctx, options.Client().ApplyURI(uri))
+
+	if err != nil {
+		panic(err)
+	}
+	defer func() {
+		if err = client.Disconnect(ctx); err != nil {
+			panic(err)
+		}
+	}()
+
+	// begin find
+	myCollection := client.Database("sample_training").Collection("zips")
+	filter := bson.D{{"pop", bson.D{{"$lte", 500}}}}
+
+	cur, err := myCollection.Find(ctx, filter)
```

Comment:
```suggestion
	cursor, err := myCollection.Find(ctx, filter)
```

Suggestion: Could this be changed to "cursor"? I think the ambiguity created by "cur" (could also mean current) is more harmful to readers than the additional three characters.

Reply:
accepted